sim/scripts/create_fixed_torso.py

```diff
@@ -30,8 +30,15 @@ def update_urdf() -> None:
                 limit.set("upper", upper)
 
     # Save the modified URDF to a new file
-    tree.write("stompy/robot_fixed.urdf")
+    tree.write("sim/stompy/robot_fixed.urdf")
 
 
 if __name__ == "__main__":
     update_urdf()
+
+
+# adjust default limits to be very constrained arouynd default standing pose
```

Comment:
Remove this: https://github.com/kscalelabs/sim/issues/21

Reply:
done